ci/cpu/build.sh

```diff
@@ -44,3 +44,32 @@ source ci/common/install-package.sh
 logger "Running AtacWorks tests..."
 cd "${WORKSPACE}"
 source ci/common/test-atacworks.sh "$WORKSPACE"
+
```

Comment:
@ntadimeti we should add here a step to `pip uninstall` the currently installed version of atacworks (develop mode). Since the `.whl` package and the source files are installed with the same version, the command in line 59  `pip install ${WORKSPACE}/atacworks_wheel/*` will actually skip the wheel file installation because this requirement is already satisfied.

Reply:
good point! Made the change

Reply:
@ntadimeti  it [doesn't seem](https://gpuci.gpuopenanalytics.com/blue/organizations/jenkins/clara-genomics%2Fgpuci%2Fatac-works%2Fprb%2Fatac-works-cpu-build/detail/atac-works-cpu-build/576/pipeline) this syntax works, you need to pass the package name explicitly.